gcsa/google_calendar.py

```diff
@@ -40,20 +41,24 @@ def __init__(self,
                 if require read only access. Default: False
         :param application_name:
                 name of the application. Default: None
+        :param token_path:
+                path to save authenticated token file. Default: "token.pickle"
```

Comment:
_Default: "token.pickle" in the same directory as the `credentials_path`_

Reply:
Also, the token file will just be loaded if it exists. So

_path to the token file to use if exists. Or path to save the authenticated token file._

(wording is discussable).

Reply:
Yeah, I originally wrote 'path to save/load authenticated token file', but that isn't really understandable if someone hasn't use any google services before.

'path to the token file to use if exists' is better, but then that doesn't communicate the user that this is where the token file will be stored after authentication flow, and they may think they need to download a token from somewhere.

Reply:
"_path to load the token file if exists or path to save it if it doesn't_" maybe?

Agree about the wordiness

Reply:
Most of the time, the place a person would really see this is the docs, so maybe its worth being that wordy, just so there isn't confusion.

What do you think about:

'existing path to load the token from, or path to save the token after authentication flow'

Theres the comment on the 'Getting Started' page that mentions what this should be used for (multiple accounts), and once the user sees authentication flow once, I think it should make sense.

Reply:
perhaps 'existing path to load the token from, or path to save the token after initial authentication flow' is better, adding 'initial'

Reply:
Makes perfect sense. And this works for me - I'd prefer wordiness to a confusion :)
I think your option is the best